pc-test-deploy-ui/src/main/resources/test/PCTestDeplymentConfiguration.xml

```diff
@@ -38,6 +38,24 @@
   <syntaxId>xwiki/2.1</syntaxId>
   <hidden>false</hidden>
   <content>{{velocity}}
+#macro (__jira $branchName)
+  #set ($patternStr = '[A-Z]++-\d++')
+  #set ($issueIdPattern = $regextool.compile($patternStr))
+  #set ($matcher = $issueIdPattern.matcher($branchName))
+  #if ($matcher.find())
+    #set ($issueId = "$!matcher.group()")
+    #set ($issueURL = "https://phenotips.atlassian.net/browse/$!{issueId}")
+    #set ($issueApiURL = "https://phenotips.atlassian.net/rest/api/3/issue/$!{issueId}?fields=summary")
+    #set ($issueData = $xwiki.getURLContent($issueApiURL))
+    #set ($issueJson = $jsontool.parse($issueData))
+    #set ($issueSummary = "$!{issueJson.fields.summary}")
+[[$issueId&gt;&gt;${issueURL}||rel="__blank" title="$!{issueSummary}" class="issue"]]
+  #set($branchName-1 = $branchName.replace($issueId, ''))
```

Comment:
Why a minus? This doesn't work in newer velocity, and since you don't use the variable anywhere else, you can just use the expression inline.

Reply:
I didn't know they're dropping support for dashes in variable names, thanks for the info. I'll get rid of the dash.